ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml

```diff
@@ -26,7 +26,7 @@ tag_specification:
   namespace: ocp
 tests:
 - as: unit
-  commands: go test ./...
+  commands: make test
```

Comment:
generally, we use `make test-unit` for unit tests... `make test` normally runs **all** tests

Reply:
Done